client/src/www/app/outline_server_repository/server.cordova.ts

```diff
@@ -23,8 +23,12 @@ export class CordovaTunnel implements PlatformTunnel {
     if (!config) {
       throw new errors.IllegalServerConfiguration();
     }
-    return pluginExecWithErrorCode<void>('start', this.id, name, config);
-  }
+    if (cordova.platformId === 'android') {
+      return pluginExecWithErrorCode<void>('start', this.id, name, config);
+    } else {
+      // TODO: implement serverName in iOS.
+      return pluginExecWithErrorCode<void>('start', this.id, config);
```

Comment:
Please merge #2075, then you can get rid of this TODO as the basic wiring was done by @fortuna in  https://github.com/Jigsaw-Code/outline-apps/pull/2075/files#diff-509f058f9661f2600b457b0d73100f9e57d98e705b7b192b63a20cebda53df6d.

Reply:
Thanks! will do.